Makefile

```diff
@@ -4,4 +4,4 @@ build:
 	go build -i -o artifacts/compete github.com/LukeJelly/Cell-Compete-Go
 
 test:
-	go test -race $(go list ./... | grep -v vendor)
```

Comment:
I'm not actually sure why I needed to change this; some of the outputs from `go list ./...` were not being passed to `go test`.